Source/ProgressionSystemRuntime/Public/Data/PSDataAsset.h

```diff
@@ -112,23 +108,18 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSDataAsset : public UMyPrimaryDataAsset
 	UPROPERTY(EditAnywhere, Category = "UI|Material", meta = (BlueprintProtected, DisplayName = "Unlocked Progression Icon"))
 	TObjectPtr<class UTexture2D> UnlockedProgressionIconInternal = nullptr;
 
-	/** Material applied for dynamic progression material (star is filled partially depends on progression) */
-	UPROPERTY(EditAnywhere, Category = "UI|Material", meta = (BlueprintProtected, DisplayName = "Dynamic Progression Material"))
-	TObjectPtr<class UMaterialInterface> DynamicProgressionMaterialInternal = nullptr;
+	/** Material applied as dynamic progression overlay material (character's bomb is filled partially depends on progression) in the main menu*/
+	UPROPERTY(EditAnywhere, Category = "UI|Material", meta = (BlueprintProtected, DisplayName = "Dynamic Progression Bomb Overlay Material"))
+	TObjectPtr<class UMaterialInterface> DynamicProgressionBombOverlayMaterialInternal = nullptr;
 
 	/** The Progression difficulty multiplier. */
 	UPROPERTY(EditDefaultsOnly, BlueprintReadOnly, meta = (BlueprintProtected, DisplayName = "Progression Multiplier Curve Table"))
 	TObjectPtr<UCurveTable> ProgressionDifficultyMultiplierCurveTableInternal = nullptr;
-
-
+	
 	/** Star Material Slot name to change the dynamic fill-in based on the progression */
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "UI|Material", meta = (BlueprintProtected, DisplayName = "Star Percent Slot Name"))
 	FName StarPercentSlotNameInternal = NAME_None;
 
-	/** Star Face Texture Parameter name to add a face texture for each character*/
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "UI|Material", meta = (BlueprintProtected, DisplayName = "Star Face Texture Parameter Name"))
-	FName StarFaceTextureParameter = NAME_None;
-
 	/** Used to tweak the stars UV. Set 1 to have no impact
 	 * Can not be 0 */
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "UI|Material", meta = (BlueprintProtected, DisplayName = "Star Material Fractional Divisor"))
```

Comment:
Please completely remove `StarMaterialFractionalDivisorInternal` property and its logic. Now it's being handled by material itself

Reply:
i did, no changes appeared as changed for some reason

Reply:
removed https://github.com/moinm3uw/ProgressionSystem/pull/66/commits/fef665989bcd9df8da189ad73c968c0dd0b010bd